### DISCUSSION TITLE
2021-11-10 Weekly Meeting 29 Agenda / Notes

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#119] [General]
Weekly Meeting 29 is happening on Wednesday November 10th, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - https://github.com/omigroup/omigroup/issues/72
     - #118 Add vision to readme (see PR #120) 
   - [glTF subgroup](https://github.com/omigroup/gltf-extensions/discussions/31)
     - Khronos Liaison 
   - WebXR Hackathon
   - https://github.com/omigroup/omigroup/issues/109
   - [website](https://github.com/omigroup/website/projects/1)
   - VKet Booth

   
 ## Add Ons
- [Call for participation: Metaversity](https://github.com/omigroup/omigroup/issues/115)
- Getting in front of the noise, together
- Twitter Spaces with Evo / Jin
  - [Jin](https://twitter.com/dankvr) hosting the first one on Friday at 2PM ET
- Pick up OMI Chan work next week?
- [Avacon](https://www.avacon.org/blog/)

**Comment by mrmetaverse** [comment]
Created this PR: https://github.com/omigroup/omigroup/pull/120 

Once added to readme and website we can move this to done -> https://github.com/omigroup/omigroup/issues/118

**Comment by mrmetaverse** [comment]
Invited to have a OMI liason with the Khronos gltf group

**Comment by mrmetaverse** [comment]
To get involved with the gLTF group: https://github.com/omigroup/gltf-extensions
